modules/hal_nordic/nrf_802154/sl_opensource/platform/nrf_802154_temperature_zephyr.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *   This file implements the thermometer abstraction that uses Zephyr sensor
+ *   API for the die termomether.
+ *
+ */
+
+#include "platform/nrf_802154_temperature.h"
+
+#include <device.h>
+#include <drivers/sensor.h>
+#include <kernel.h>
+#include <stdbool.h>
+
+/** @brief Value of the last temperature measurement. */
+static int8_t m_temperature;
```

Comment:
Please don't use `m_` prefix in Zepyr tree.

Reply:
Fixed